src/creates/actor_run.js

```diff
@@ -21,24 +21,29 @@ const runActor = async (z, bundle) => {
             'Content-Type': inputContentType,
         };
     }
-    if (inputBody) requestOpts.body = inputBody;
-
-    const runResponse = await wrapRequestWithRetries(z.request, requestOpts);
-
-    let run = runResponse.json;
-    if (runSync) {
-        run = await enrichActorRun(z, run);
+    if (inputBody) {
+        if (inputContentType.includes('application/json')) {
+            try {
+                JSON.parse(inputBody);
+            } catch (err) {
+                throw new Error('Please check that your input body is valid JSON.');
```

Comment:
is **a** valid JSON

Reply:
Shouldn't we say "Your input is not a valid JSON: [errorMsg]" as we can be sure at this point that it's not valid?

Reply:
The Zapier team wants to make error messages as simple as possible to BFU.